src/interactive.sh

```diff
@@ -38,3 +38,94 @@ function check_ssh_installation()
   [[ -n "$cmd" ]]
 
 }
+
+function interactive_init()
+{
+
+  local cmd=''
+  local distro=''
+  local has_ssh=0
+  local ssh_keys
+  local key_name ssh_dir
+  local ssh_user ssh_ip
+
+  load_string "$KW_LIB_DIR/strings/init.txt"
```

Comment:
You must handle the return of load_string because it can fail. Additionally, you changed this function name in the previous commit, i.e., load_string does not exist anymore :/
Did you test this PR?